src/yb/master/master_main.cc

```diff
@@ -69,6 +71,14 @@ static int MasterMain(int argc, char** argv) {
   // Reset some default values before parsing gflags.
   FLAGS_rpc_bind_addresses = strings::Substitute("0.0.0.0:$0", kMasterDefaultPort);
   FLAGS_webserver_port = kMasterDefaultWebPort;
+
+  string host_name;
+  if (GetHostname(&host_name).ok()) {
+    FLAGS_metric_node_name = strings::Substitute("$0:$1", host_name, kMasterDefaultWebPort);
+  } else {
+      LOG(INFO) << "Failed to get master's host name, keeping default metric_node_name";
```

Comment:
minor: fix indentation

Reply:
Should I align it with the else statement?

Reply:
Used this guideline: https://google.github.io/styleguide/cppguide.html#Conditionals